include/services/OswServiceTasks.h

```diff
@@ -14,9 +14,7 @@ extern OswServiceTaskBLECompanion bleCompanion;
 //extern OswServiceTaskExample example;
 extern OswServiceTaskWiFi wifi;
 extern OswServiceTaskWebserver webserver;
-#ifndef NDEBUG
```

Comment:
What's the difference between yes and no?

Reply:
If `NDEBUG` is defined (= no debug) we remove a lot of console logging and debug statements (like the debug app). `#ifndef` basically tells "if not defined".